packages/aragon-cli/src/commands/dao_cmds/new.js

```diff
@@ -10,6 +10,7 @@ const {
   getRecommendedGasLimit,
   parseArgumentStringIfPossible,
 } = require('../../util')
+const kernelAbi = require('@aragon/os/build/contracts/Kernel.json').abi
```

Comment:
Very small detail but I would remove the `.json` extension to keep consistency with other imports.